### ISSUE TITLE
[Feat] Toggle Streamer mode

### ISSUE THREAD

**Issue by RavenX8** [bitfocus/companion-module-discord-api#25]
**Describe the feature**
I would like to be able to toggle streamer mode on and off. This should also include a feedback with the status of discords streamer mode

**Usecases**
Sometimes I would like to mute all notifications without deafening myself. Also I do stream so I need to pop into discord and toggle the setting each time.

**Comment by thedist** [comment]
I'll try looking in to this, but the so far it appears the API documentation still has no notion of the streamer mode. With Discord themselves neglecting RPC/IPC it is entirely possible that this may not be possible.

**Comment by RavenX8** [comment]
Yeah, I just checked the discord dev docs and I defiantly do not see anything where you can toggle this setting. Thanks anyway.
